lib/celluloid/groups/proactor.rb

```diff
@@ -0,0 +1,56 @@
+require 'thread'
+
+module Celluloid
+  class Group
+    class Proactor < Group
```

Comment:
Well there is thread creation, but it's sort of a high integrity thing going on here:

0. Create thread.
0. Small to large  ( possibly blocking ) activity.
0. Conclude, blocking.

I focus on 2 & 3 more, and how that's handled, trying to fit a pattern, otherwise I'd just call it Dispatcher and call it good.